src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -853,11 +963,28 @@ void TableDockWidget::setGroupLabel(char label) {
       QVariant v = item->data(0, Qt::UserRole);
       PeakGroup *group = v.value<PeakGroup *>();
       if (group != NULL) {
+        if (!(group->label=='g'||group->label=='b')){
+          numberOfGroupsMarked+=1;
+          if (numberOfGroupsMarked == 1) curationTimer.start();
```

Comment:
the timer was being used to calculate the time saved by using the ML model. This was presented on the toast you just removed. We can safely remove this timer now.

Reply:
Along with the tenPeaksMarked signal.